sycl/test/basic_tests/parallel_for_disable_range_roundup_syclstd.cpp

```diff
@@ -0,0 +1,51 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple -sycl-std=2017 %s -o %t.out
+// RUN: env SYCL_PARALLEL_FOR_RANGE_ROUNDING_TRACE=1 %RUN_ON_HOST %t.out
+
+#include <CL/sycl.hpp>
+
+using namespace sycl;
+
+range<1> Range1 = {0};
+
+void check(const char *msg, size_t v, size_t ref) {
+  std::cout << msg << v << std::endl;
+  assert(v == ref);
+}
+
+int try_rounding_off(size_t size) {
+  range<1> Size{size};
+  int Counter = 0;
+  {
+    buffer<range<1>, 1> BufRange(&Range1, 1);
+    buffer<int, 1> BufCounter(&Counter, 1);
```

Comment:
It seems the tests need to be moved to https://github.com/intel/llvm-test-suite.

Reply:
Does this mean two separate PRs, one for the SYCL header change in intel/llvm and one for the tests in intel/llvm-test-suite? How to synchronize the two changes?

Reply:
@vladimirlaz Could you please help answering?

Reply:
Tests combined into one, and moved to test repo.